docs/application/dotnet/guides/nui/nui-components/Toast.md

```diff
@@ -0,0 +1,40 @@
+# Toast
+Toast is a common component and provides simple messages.  
+Unlike other popups, a toast consists only of a body field as it is just used for providing simple feedback to the user’s actions.  
+You can use a toast to provide simple messages when the user does not need to make an additional action or confirmation.  
+A toast will automatically disappear after a certain time.
+
+![Toast](./media/toast.png)
+
+
+## Create with property
+To create a toast using property, follow these steps:
+1. Create toast using the default constructor:
+
+```

```

Comment:
Kindly check the global comment for code block language identifier and improper alignment with bullet point.